spec/std/big_rational_spec.cr

```diff
@@ -0,0 +1,148 @@
+require "spec"
+require "big_rational"
+
+def br(n, d)
```

Comment:
Better to make this `private def br` so it doesn't conflict with other methods in the future. Same for `test_comp`. See http://crystal-lang.org/docs/syntax_and_semantics/visibility.html#private-top-level-methods